android/airship-framework-proxy/src/main/java/com/urbanairship/android/framework/proxy/AirshpPluginExtender.kt

```diff
@@ -19,4 +20,19 @@ public interface AirshipPluginExtender {
      * @param airship The airship instance.
      */
     public fun onAirshipReady(context: Context, airship: UAirship)
+
+    /**
+     * Used to extend the AirshipConfig. The configBuilder will have the default config applied from the properties file if available,
+     * any config defined by the module.
+     * @param context The application context.
+     * @param configBuilder The config builder
+     * @return The config builder.
+     */
+    public fun extendConfig(
```

Comment:
Instead of making this a breaking change we could add a new interface that the pluginExtender could optionally implement. That makes it a little harder to use though